TestApps/Sasquatch/Sasquatch/Base.lproj/LaunchScreen.storyboard

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="13529" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" launchScreen="YES" useTraitCollections="YES" colorMatched="YES" initialViewController="D7o-BS-rls">
+    <device id="retina5_9" orientation="portrait">
+        <adaptation id="fullscreen"/>
+    </device>
+    <dependencies>
+        <deployment identifier="iOS"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="13527"/>
+        <capability name="Alignment constraints with different attributes" minToolsVersion="5.1"/>
+        <capability name="Aspect ratio constraints" minToolsVersion="5.1"/>
+        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
+    </dependencies>
+    <scenes>
+        <!--View Controller-->
+        <scene sceneID="Ivr-2Z-SHp">
+            <objects>
+                <viewController id="D7o-BS-rls" sceneMemberID="viewController">
+                    <layoutGuides>
+                        <viewControllerLayoutGuide type="top" id="tYG-Rd-uMs"/>
+                        <viewControllerLayoutGuide type="bottom" id="0VW-kH-3Dp"/>
+                    </layoutGuides>
+                    <view key="view" contentMode="scaleToFill" id="pYN-Po-Jnv">
+                        <rect key="frame" x="0.0" y="0.0" width="375" height="812"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <subviews>
+                            <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" textAlignment="center" lineBreakMode="tailTruncation" numberOfLines="2" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="ASe-Tr-X1Z">
+                                <rect key="frame" x="57" y="456" width="261" height="41"/>
+                                <constraints>
+                                    <constraint firstAttribute="height" constant="41" id="rb3-0k-9WO"/>
+                                </constraints>
+                                <string key="text">Starting Visual Studio App Center
+Puppet Test Application
```

Comment:
Format

Reply:
This is not Puppet app. Change the name to Sasquatch.
It was initially displaying Swift/Objective C and this is why there are two launch storyboard.
I don't care of combining those, FYI.